lib/components/Input/text-field-with-steppers/text-field-with-steppers.scss

```diff
@@ -1,19 +1,19 @@
 .quill-input-steppers {
-    &-decrement {
-        padding: var(--core-spacing-400);
+    &-button {
+        border-radius: var(--core-borderRadius-500);
         cursor: pointer;
         display: flex;
         margin: auto;
-    }
-    &-increment {
-        padding: var(--core-spacing-400);
-        cursor: pointer;
-        display: flex;
-        margin: auto;
-        margin-left: var(--core-spacing-400);
-    }
-    &--disabled {
-        opacity: var(--core-opacity-600);
-        cursor: not-allowed;
+        padding: var(--core-spacing-300) var(--core-spacing-500);
+
+        &:hover:not(:disabled) {
+            background-color: var(--core-color-opacity-black-100);
+        }
+        &:active:not(:disabled) {
+            background-color: var(--core-color-opacity-black-200);
+        }
+        &:disabled {
+            cursor: not-allowed;
+        }
     }
 }
```

Comment:
just a nit-pick:) Do you mind adding a line break at the end of the file to stop Git from complaining about it with ⊖ sign?

Reply:
Sure, added